dotCMS/src/main/java/com/dotcms/graphql/util/TypeUtil.java

```diff
@@ -34,17 +38,22 @@ public static GraphQLObjectType createObjectType(final String typeName, final Ma
 
     public static GraphQLObjectType createObjectType(final String typeName,
             final Map<String, TypeFetcher> fieldsTypesAndFetchers) {
+
         final GraphQLObjectType.Builder builder = GraphQLObjectType.newObject().name(typeName);
 
-        fieldsTypesAndFetchers.keySet().forEach((key)->{
-            builder.field(newFieldDefinition()
-                    .name(key)
-                    .type(fieldsTypesAndFetchers.get(key).getType())
-                    .dataFetcher(fieldsTypesAndFetchers.get(key).getDataFetcher()!=null
-                            ?fieldsTypesAndFetchers.get(key).getDataFetcher()
-                            :new PropertyDataFetcher<String>(key))
-            );
-        });
+        for (String key : fieldsTypesAndFetchers.keySet()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'key' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5685344)